updates/work-in-progress/220118_Note.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`work-in-progress` folder name should be replaced with the next available database version number.